p2/index.qmd

```diff
@@ -32,27 +33,28 @@ Install the packages by typing in the following commands into RStudio (you do no
 
 If you need to install any of these packages use:
 
-::: {.panel-tabset group="language"}
+::: {.panel-tabset}
```

Comment:
Why not 'language'?

Reply:
When this option is set, clicking on the Python tab will switch all code blocks to Python. However, due to the different lengths of the R and Python code blocks, this causes the content within the current browser window to suddenly jump, creating significant interference with reading. You have to scroll up or down to find the section you were just reading. Students using Python still need to compare the results of the R code execution, so such page jumps and interruptions can happen frequently. Removing the language grouping can avoid this problem, as each code block switch would only apply to the current tab, resulting in less disruption to the reading experience - in my opinion, this has a better user experience.

Reply:
OK, great explanation, thanks!